plugins/modules/files/sapcar_extract.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2021, Rainer Leber <rainerleber@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: sapcar_extract
+short_description: Manages SAPCAR from SAP
+version_added: "3.2.0"
+description:
+    - Provides support for unpacking sar/car files with the SAPCAR binary from SAP and pulling
+      information back into Ansible.
+options:
+  path:
+    description: The path to the SAR/CAR file.
+    type: path
+    required: true
+  dest:
+    description:
+      - The destination where SAPCAR extracts the SAR file. Missing folders will be created.
+        If this parameter is not provided it will unpack in the same folder as the SAR file.
+    type: path
+  binary_path:
+    description:
+      - The path to the SAPCAR binary e.g. /home/dummy/sapcar or https://myserver/SAPCAR.
+        If this parameter is not provided the module will try excute from application binary e.g /usr/local/bin.
+    type: path
+  signature:
+    description:
+      - If true the Signature will be extracted
```

Comment:
```suggestion
      - If C(true) the signature will be extracted
```
In which form will it be extracted?